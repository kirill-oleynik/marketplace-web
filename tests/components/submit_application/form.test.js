const React = require('react');
const { shallow } = require('enzyme');
const toJSON = require('enzyme-to-json').default;
const SubmitApplicationForm = require('../../../app/components/submit_application/form').default;

describe('#render', () => {
  test('it renders correctly', () => {
    const component = shallow(
      <SubmitApplicationForm
        t={() => {}}
        errors={{}}
        onSubmit={() => {}}
        revalidation={{}}
        closeModal={() => {}}
      />
    );

    expect(toJSON(component)).toMatchSnapshot();
  });
});
