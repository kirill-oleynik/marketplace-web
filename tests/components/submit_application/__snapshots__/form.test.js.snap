// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#render it renders correctly 1`] = `
<SubmitApplicationForm
  closeModal={[Function]}
  errors={Object {}}
  onSubmit={[Function]}
  revalidation={
    Object {
      "UPDATE_FIELD": "UPDATE_FIELD",
      "VALIDATE": "VALIDATE_ALL",
      "asyncErrors": Object {},
      "debounce": Object {},
      "errors": Object {},
      "form": Object {},
      "onChange": [Function],
      "onSubmit": [Function],
      "settings": Object {
        "validateOnChange": false,
        "validateSingle": false,
      },
      "submitted": false,
      "updateAsyncErrors": [Function],
      "updateErrors": [Function],
      "updateState": [Function],
      "updateValue": [Function],
      "updateValueAndValidate": [Function],
      "valid": false,
      "validateValue": [Function],
    }
  }
  t={[Function]}
/>
`;
